code/modules/mob/new_player/sprite_accessories/ipc/ipc_optics.dm

```diff
@@ -1,26 +1,26 @@
 /datum/sprite_accessory/body_markings/head/optics
-    icon = 'icons/mob/sprite_accessories/ipc/ipc_optics.dmi'
-    name = "Humanoid Optics"
-    species_allowed = list("Machine")
-    icon_state = "optics"
-    models_allowed = list("Bishop Cybernetics", "Hesphiastos Industries", "Ward-Takahashi", "Xion Manufacturing Group", "Zeng-Hu Pharmaceuticals") //Should be the same as the manufacturing company of the limb in robolimbs.dm
+	name = "Humanoid Optics"
+	icon = 'icons/mob/sprite_accessories/ipc/ipc_optics.dmi'
+	icon_state = "optics"
+	species_allowed = list("Machine")
+	models_allowed = list("Tableau Entertainment Powered Mechanics", "Bishop Cybernetics", "Hesphiastos Industries", "Ward-Takahashi", "Xion Manufacturing Group", "Zeng-Hu Pharmaceuticals", "Zeng-Hu Spirit") //These 'monitors' need eyes, I mean, uh, "optics".
 
 /datum/sprite_accessory/body_markings/head/optics/bishop_alt
-	name = "Bishop Alt. Optics"
+	name = "Bishop Cybernetics Optics"
 	icon_state = "bishop_alt_optics"
-	models_allowed = list("Bishop Cybernetics alt.")
+	models_allowed = list("Bishop Knight")
```

Comment:
Bishop Knight, Bishop Rook, Bishop Cybernetics, Bishop Castle - oh come on! You have to admit that sounds cooler 🙄

Reply:
This file is the one that tells you which monitors need optics (eyes sprites) added. You may notice one of the ported monitor sprites is in there. I compared the dmi files.